shawn-patel.txt

```diff
@@ -0,0 +1,171 @@
+
+LINK TO PART 2   CODE PEN:
+https://codepen.io/ShawnP96/pen/eYmmBqm?editors=1100
+https://codepen.io/ShawnP96/pen/eYmmBqm?editors=1100
+https://codepen.io/ShawnP96/pen/eYmmBqm?editors=1100
+
+
+
+
+
+
+
+<!-- reveiw questions to Submit in today's file firt-lastname.txt --> 
+<!--
+Copy the questions below into the firt-lastname.txt file on your github repo - answer each question and then push your changes. 
+
+    1. What is Semantic HTML? 
+        Semantic HTML is using understandable elements/code
+    
+    2. What is HTML used for?
+        HTML is used to create the page, while css is used to decorate the page.
+
+    3. What is an attribute and where do we put it? 
+     attribute's are name value pairs and we put can put it to create a link (a href="#") or to create a class and then later call it in css (div class="about")
+    
+    4. What is the h1 tag used for? How many times should I use it on a page?
+        h1 is used for the most important text, so a title. we can technically call it as many times as we want but it should be reserved for utmost important words
+    
+    5. Name two tags that have required attributes
+
+    6. What do we put in the head of our HTML document? 
+        many things can go into the <head></head>, the title
+    
+    7. What is an id? 
+        it is a type of css selector with the highest prio, but reserved for anchor tags
+    
+    8. What elements can I add an id to? 
+        anything you want the highest prio too, and for links/anchor tags
+   
+    9. How many times can I use the same id on a page? 
+        once per unique id
+   
+    10. What is a class?
+    a css selector with a higher priority (.class) 
+    
+    11. What elements can I add a class to? 
+        anything you want to add a specific style too
+   
+    12. How many times can I use the same class on a page? 
+    as many times as you want however the the css property will be applied to element the class is on.
+    
+    13. How do I get my link to open in a new tab?
+    add target="_blank to your links/anchor tags
+    
+    14. What is the alt attribute used for? 
```

Comment:
Here is more information on the alt attribute https://www.w3schools.com/tags/att_img_alt.asp